instant/src/main/java/com/adyen/checkout/instant/InstantPaymentComponent.kt

```diff
@@ -74,5 +75,21 @@ class InstantPaymentComponent internal constructor(
 
         @JvmField
         val PROVIDER = InstantPaymentComponentProvider()
+
+        /**
+         * These are the payment method types known to be supported by the [InstantPaymentComponent]. There are
+         * payment method types that are not listed here that are actually supported. See
+         * [InstantPaymentComponentProvider.isPaymentMethodSupported] for more details.
+         */
+        @JvmField
+        val PAYMENT_METHOD_TYPES = listOf(
```

Comment:
Even with the comment explaining it I'm not sure about this list being public. From an external perspective I don't see a reason why it should be part of the public API since it has no use for someone else using the library

Reply:
I do agree with you, but other components also have this field public and imo it could be private for those as well. I did this out of consistency

Reply:
What if we put this list somewhere completely different? it doesn't have to have the same name or even be public

Reply:
Sure, let me find a nice place

Reply:
I like the idea of extracting this logic. Shall we merge this PR first, so we can work on it in your PR? Or you think it's better if I do it here?

Reply:
Sure, let's merge this first. We can discuss the other approach later